network-protection/network-protection-impl/src/main/java/com/com/duckduckgo/networkprotection/subscription/NetpSubscriptionManager.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.com.duckduckgo.networkprotection.subscription
+
+import com.com.duckduckgo.networkprotection.subscription.NetpSubscriptionManager.VpnStatus
+import com.com.duckduckgo.networkprotection.subscription.NetpSubscriptionManager.VpnStatus.ACTIVE
+import com.com.duckduckgo.networkprotection.subscription.NetpSubscriptionManager.VpnStatus.EXPIRED
+import kotlinx.coroutines.flow.Flow
+
+// TODO this interface is moved here temporarily and its implementation remains in network-protection-subscription-internal
+// to expedite Privacy Pro launch, we'll clean up later on
+interface NetpSubscriptionManager {
```

Comment:
Moved just the interface to avoid the noise in the PR result of moving code from `network-protection-subscriptions-internal` module to `network-protection-impl`.
But will probably do this in a stacked PR later on